src/solid/scope.ts

```diff
@@ -11,12 +11,12 @@ import {useStoreMapBase, useUnitBase} from './lib/base'
 import {useGate as commonUseGate, createGateImplementation} from './lib/gate'
 import type {Gate} from './index.h'
 
-const ScopeContext = createContext<Scope | null>(null)
+const ScopeContext = createContext(null as Scope | null)
 export const {Provider} = ScopeContext
 
-function getScope() {
+export function getScope(forceScope?: boolean) {
```

Comment:
In my opinion, it is better not to export this function from public module, let's move it somewhere inside the package?

Reply:
Maybe take move everything the file `scope.ts` into an `ssr.ts` file. Leave only public methods in the `ssr.ts`.

Reply:
Yep, that would be better move 🤔 

`getScope` should not be public, that's for sure